internal/ld/ld.go

```diff
@@ -92,14 +93,22 @@ func InitApiClient(options ApiOptions) ApiClient {
 
 func (c ApiClient) GetFlagKeyList() ([]string, error) {
 	ctx := context.WithValue(context.Background(), ldapi.ContextAPIKey, ldapi.APIKey{Key: c.Options.ApiKey})
-	flags, _, err := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, c.Options.ProjKey, nil)
+
+	flags, _, err := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, c.Options.ProjKey, &ldapi.GetFeatureFlagsOpts{Summary: optional.NewBool(true)})
```

Comment:
What's the purpose of `Summary: optional.NewBool(true)`?

Reply:
Ah, should have noted this change. This filter option will make the api requests for feature flags a big faster by omitting flag configuration data that `ld-find-code-refs` doesn't need

Reply:
That makes sense. Thanks!